projects/control-service/projects/job-builder-secure/Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN apk add --no-cache --update \
     git \
     bash
 
-RUN apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.10/main python3=3.10.9-r1 py3-pip \
+RUN apk add --update --no-cache python3=3.10.10-r0 py3-pip \
```

Comment:
It was already on 3.10 I just upgraded to the latest version of 3.10.
I didn't want to change the  python version because I thought it might have been chosen for a specific reason.